pkg/machine/fedora.go

```diff
@@ -69,6 +75,30 @@ func (f FedoraDownload) HasUsableCache() (bool, error) {
 	return info.Size() == f.Size, nil
 }
 
+func (f FedoraDownload) CleanCache() error {
```

Comment:
This is the same as  (f FcosDownload) CleanCache(), use the same code instead of duplicating